src/components/Layout/components/useMediaQuery.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {MEDIA_TO_VALUE} from '../constants';
+import {MediaPartial, MediaProps} from '../types';
+
+export const mockMediaQueryList: MediaQueryList = {
+    media: '',
+    matches: false,
+    onchange: () => {},
+    addListener: () => {},
+    removeListener: () => {},
+    addEventListener: () => {},
+    removeEventListener: () => {},
+    dispatchEvent: (_: Event) => true,
+};
+
+const MEDIA_QUERIES: MediaProps<string> = {
+    mobile: `(max-width: ${MEDIA_TO_VALUE.mobile}px)`,
+    tablH: `(min-width: ${MEDIA_TO_VALUE.mobile + 1}px) and (max-width: ${MEDIA_TO_VALUE.tablH}px)`,
+    lptpS: `(min-width: ${MEDIA_TO_VALUE.tablH + 1}px) and (max-width: ${MEDIA_TO_VALUE.lptpS}px)`,
+    lptpM: `(min-width: ${MEDIA_TO_VALUE.lptpS + 1}px) and (max-width: ${MEDIA_TO_VALUE.lptpM}px)`,
+    dsktp: `(min-width: ${MEDIA_TO_VALUE.lptpM + 1}px) and (max-width: ${MEDIA_TO_VALUE.dsktp}px)`,
```

Comment:
is `dsktp` is really much shorter than `desktop`?  I think second option looks better, for tabl and lptp too

Reply:
Are you talking about renaming these variables?
like:
`dsktp` -> `desktop`
`lptpM` -> `laptopM`
`tablH` -> `tablet`

Reply:
yep)